### DISCUSSION TITLE
Exiting out of the Editor Test whilst the beatmap is still playing will play the editor itself.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#15620] [Q&A] [answered]
Steps to reproduce:
1. Enter into a beatmap editor.
2. go to any time in the beatmap
3. Press the "Test" button
4. Press `Escape` whilst the song is still playing.

Expected:
Editor would be paused at the time you were currently at.

Observed:
Editor would play from the time you were currently at.

**Comment by Coppertine** [comment, marked as answer]
Self answering as to be done in #15609

**Reply by bdach** [reply]
that pull does almost the thing you want it to, except that the playback does not resume on returning to editor. do you prefer that were the case? if so can you elaborate why?

**Reply by Coppertine** [reply]
Expected to not resume when returning to editor, so the pull request is good!
